tck/src/test/java/org/reactivestreams/tck/SubscriberBlackboxVerificationTest.java

```diff
@@ -112,6 +114,34 @@ public void required_spec205_blackbox_mustCallSubscriptionCancelIfItAlreadyHasAn
   }
 
   @Test
+  public void required_spec205_blackbox_mustCallSubscriptionCancelIfItAlreadyHasAnSubscriptionAndReceivesAnotherOnSubscribeSignal_shouldGetCompletion() throws Throwable {
+    final CountDownLatch completion = new CountDownLatch(1);
+      
+    customSubscriberVerification(new KeepSubscriptionSubscriber() {
+      volatile Subscription sub;
+
+      @Override
+      public void onSubscribe(Subscription s) {
+        super.onSubscribe(s);
+        if (sub != null) {
+          sub = s;
+          s.request(1);
+        } else {
+          // the second one we cancel
+          s.cancel();
+        }
+      }
+
+      @Override
+      public void onComplete() {
+        completion.countDown();
+      }
+    }).required_spec205_blackbox_mustCallSubscriptionCancelIfItAlreadyHasAnSubscriptionAndReceivesAnotherOnSubscribeSignal();
+    
+    completion.await(1, TimeUnit.SECONDS);
```

Comment:
Ok, yes it's enough.
It does not matter for different impls, this provides the impl and tests the tck, no different impls will be provided for this codepath.